code_schemes/issues_in_school.json

```
@@ -3,7 +3,62 @@
   "Name": "Issues in School",
   "Version": "0.0.0.1",
   "Codes": [
-
+    {
+      "CodeID": "code-0aefd71e",
+      "CodeType": "Normal",
+      "DisplayText": "curriculum challenges",
+      "NumericValue": 1,
+      "StringValue": "curriculum_challenges",
+      "VisibleInCoda": true
+    },
+    {
+      "CodeID": "code-5e38906f",
+      "CodeType": "Normal",
+      "DisplayText": "unqualified teachers",
+      "NumericValue": 2,
+      "StringValue": "unqualified_teachers",
+      "VisibleInCoda": true
+    },
+    {
+      "CodeID": "code-8a025741",
+      "CodeType": "Normal",
+      "DisplayText": "negative impact of social media",
+      "NumericValue": 3,
+      "StringValue": "negative_impact_of_social_media",
+      "VisibleInCoda": true
+    },
+    {
+      "CodeID": "code-02002816",
+      "CodeType": "Normal",
+      "DisplayText": "student lack of motivation",
```

```suggestion
      "DisplayText": "students' lack of motivation",
```
(for optional consistency with the next code)